RHEL/7/input/xccdf/services/ssh.xml

```diff
@@ -395,6 +461,29 @@ can allow an attacker to move trivially to other hosts.
 <tested by="DS" on="20121024"/>
 </Rule>
 
+<Rule id="enable_x11_forwarding" severity="high">
+<title>Enable X11 Fordwarding</title>
```

Comment:
To align with DISA's intent, lets change the title to "Enable Encrypted X11 Forwarding"